trade-finance-manager-ui/server/controllers/case/cancellation/reason-for-cancelling.controller.ts

```diff
@@ -3,19 +3,51 @@ import { CustomExpressRequest } from '@ukef/dtfs2-common';
 import { PRIMARY_NAVIGATION_KEYS } from '../../../constants';
 import { asUserSession } from '../../../helpers/express-session';
 import { ReasonForCancellingViewModel } from '../../../types/view-models';
+import { validateReasonForCancelling } from './validation/validate-reason-for-cancelling';
+
+export type GetReasonForCancellingRequest = CustomExpressRequest<{ params: { _id: string } }>;
+export type PostReasonForCancellingRequest = CustomExpressRequest<{ params: { _id: string }; reqBody: { 'reason-for-cancelling': string | undefined } }>;
 
 /**
  * controller to get the reason for cancelling page
  */
-export const getReasonForCancelling = (req: CustomExpressRequest<{ params: { dealId: string } }>, res: Response) => {
-  const { dealId } = req.params;
+export const getReasonForCancelling = (req: GetReasonForCancellingRequest, res: Response) => {
+  const { _id } = req.params;
   const { user } = asUserSession(req.session);
 
   const reasonForCancellingViewModel: ReasonForCancellingViewModel = {
     activePrimaryNavigation: PRIMARY_NAVIGATION_KEYS.ALL_DEALS,
     user,
     ukefDealId: '0040613574', // TODO: DTFS2-7350 get values from database
```

Comment:
Appreciate not in your PR, buw where did this number come from and why is it set to this deal number please.

Reply:
This is stub data with the TODO to get this from the database in a future ticket.